mteb/tasks/Classification/ara/TweetEmotionClassification.py

```diff
@@ -44,3 +45,38 @@ def dataset_transform(self):
         self.dataset = self.stratified_subsampling(
             self.dataset, seed=self.seed, splits=["train"]
         )
+
+
+class TweetEmotionClassificationV2(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="TweetEmotionClassification.v2",
+        dataset={
+            "path": "mteb/tweet_emotion",
+            "revision": "0a5ed32f2f7ab21271de1323e6f3daaedf6c7874",
+        },
+        description="A dataset of 10,000 tweets that was created with the aim of covering the most frequently used emotion categories in Arabic tweets.",
```

Comment:
same + numbers probably don't match

Reply:
It's funny that 10,000 is also an incorrect number, as there were originally 10,065 rows